### PR TITLE
Fix #432 (url translation)

### LINKED ISSUE #432: Provide proper translations for URL patterns
### Expected Behaviour

URL patterns are translated to French e.g. /order/list becomes /commande/liste. 
### Actual Behaviour

Currently, French translations for URL patterns are the exact same values as their English counterparts. Also, it's possible not all patterns are translated in the project.
### Steps to reproduce
- Set your first accepted language to French (in Chrome, make sure French is the first item in chrome://settings/languages)
- See that the URLs are in French. 
### Notes

Translations can be changed, but the developer should also make sure that all url patterns are translatable via ugettext_lazy as explained [in the Django docs](https://docs.djangoproject.com/en/dev/topics/i18n/translation/#url-internationalization)

### PR DESCRIPTION
**Pull request by lingxiaoyang**
### Changes proposed in this pull request:

* Add translatable strings in all `urls.py`.
* Add language prefix `/fr/` in URL for French. 
* Add translatable URL name for client creation form wizard.
* Add new test setting file, for running the unit test in French.
* Fix problems in French unit test.
* Add French unit test to Travis CI configuration.

### Status

- [x] READY
- [ ] HOLD: need rebase after merging PR #601 
- [ ] WIP (Work-In-Progress)

### How to verify this change

1. Visit `http://localhost:8000/fr/`, it should be French.
2. Visit every page, the URLs should be in French now (not all, because they need translation in `.po` files)
3. Go to client creation form wizard: the URL should show the steps in French.
4. Execute French unit test by: 
````
python3 src/manage.py test --settings=sous_chef.settings_test_fr
````
5. Go to Travis CI: the build log should include the French unit test.

Verify English:
1. Visit `http://localhost:8000`, it should be in English by default.
2. Now the URLs should be all in English

### Deployment notes and migration

none

### New translatable strings

- [x] If applicable, I have included updated .po files with new strings (see http://goo.gl/kfBO7N)

### Additional notes

none

### CODE REVIEW

File: src/member/urls.py [hunk 1]
```diff
@@ -83,24 +94,27 @@
...
 # Handle client update forms URL
 for k, v in member_update_forms:
+    with translation.override('en'):
+        internal_name = str(k)
     urlpatterns.append(
-        url(_(r'^(?P<pk>\d+)/update/{}/$'.format(k)), v.as_view(),
-            name='member_update_' + k)
+        url(string_concat(_(r'^(?P<pk>\d+)/update/'), k, '/$'),
+            v.as_view(),
+            name='member_update_' + internal_name)
```
Comment (erozqba) [inline src/member/urls.py]: The "internal_name" part of the route get translated, but no the 'member_update_' so we can have named URL like 'member_update_information_basique'.
You could change the implementation to translate the URL but keep route names in English?

**Comment by lingxiaoyang** [comment]
I will need to provide the French URLs based on the community's translation since one month. (For easier testing)

File: src/member/urls.py (hunk 1 above)
Comment (lingxiaoyang) [inline src/member/urls.py]: @erozqba  I improved this part. In fact, I think the route names were in English but the code was a bit confusing... Please check out the latest commit.

**Comment by lingxiaoyang** [comment]
@erozqba Sure! I'll let you know - this will need to be rebased after merging #664.

**Comment by aaboffill** [comment]
@lingxiaoyang I think that you have some voluntary work :D ... with conflicts...

**Comment by lingxiaoyang** [comment]
@aaboffill Thank you! I'll update it.

**Comment by aaboffill** [comment]
@lingxiaoyang  By experience, maybe you need to wait to the last moment to fix the conflicts, there are several PR per week and it's very probably that you will have conflicts again before merge :)

**Comment by lingxiaoyang** [comment]
@aaboffill  That's true. I'll now clean this PR a bit and squashing some commits...

**Comment by erozqba** [comment]
:+1:

**Pull request merged**